bundles/org.openhab.binding.netatmo/src/test/java/org/openhab/binding/netatmo/internal/discovery/NetatmoModuleDiscoveryServiceTest.java

```diff
@@ -0,0 +1,278 @@
+import org.eclipse.smarthome.core.i18n.TranslationProvider;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ThingUID;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.openhab.binding.netatmo.internal.handler.NetatmoBridgeHandler;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * @author Sven Strohschein - Initial contribution
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class NetatmoModuleDiscoveryServiceTest {
+
+    private NetatmoModuleDiscoveryServiceAccessible service;
+    private NetatmoBridgeHandler bridgeHandlerSpy;
+
+    @Before
+    public void before() {
+        Bridge bridgeMock = mock(Bridge.class);
+        when(bridgeMock.getUID()).thenReturn(new ThingUID("netatmo", "bridge"));
+
+        bridgeHandlerSpy = spy(new NetatmoBridgeHandler(bridgeMock, null));
+
+        LocaleProvider localeProviderMock = mock(LocaleProvider.class);
+        TranslationProvider translationProvider = mock(TranslationProvider.class);
+
+        service = new NetatmoModuleDiscoveryServiceAccessible(bridgeHandlerSpy, localeProviderMock, translationProvider);
+    }
+
+    @Test
+    public void testStartScanNothingActivated() {
+        service.startScan();
+
+        assertEquals(0, service.getDiscoveredThings().size());
+    }
+
+    @Test
+    public void testStartScanDiscoverWeatherStationNoStationsBody() {
+        activateDiscoveryWeatherStation();
+
+        service.startScan();
+
+        assertEquals(0, service.getDiscoveredThings().size());
+    }
+
+    @Test
+    public void testStartScanDiscoverWeatherStationNoStations() {
+        activateDiscoveryWeatherStation();
+
+        when(bridgeHandlerSpy.getStationsDataBody(null)).thenReturn(Optional.of(new NAStationDataBody()));
+        service.startScan();
+
+        assertEquals(0, service.getDiscoveredThings().size());
+    }
+
+    @Test
+    public void testStartScanDiscoverWeatherStationNoStationName() {
+        recordStationBody(createStation());
+
+        service.startScan();
+
+        List<DiscoveryResult> discoveredThings = service.getDiscoveredThings();
+        assertEquals(1, discoveredThings.size());
+        //Expected is only the type name, because a station name isn't available
+        assertEquals("NAMain", discoveredThings.get(0).getLabel());
+    }
+
+    @Test
+    public void testStartScanDiscoverWeatherStation() {
+        NAMain station = createStation();
+        station.setStationName("Neu Wulmstorf");
+
+        recordStationBody(station);
+
+        service.startScan();
+
+        List<DiscoveryResult> discoveredThings = service.getDiscoveredThings();
+        assertEquals(1, discoveredThings.size());
+        //Expected is the type name + station name, because both is available
```

Comment:
Both are... 
Space before "Expected"

Reply:
ok, I have now executed the Maven goal spotless:apply. There were a few other files which had formatting errors. Now spotless:check is also running successfully. :-)